caproto/ioc_examples/lakeshore.py

```diff
@@ -0,0 +1,399 @@
+"""
+An IOC with a simulated temperature controller.
+This demonstrates how to do put_completion with a caproto ioc and ophyd device.
+
+Another interesting part of this example is the use of settle_time.
+Settle_time allows us to wait an additional amount after the put_completion.
+It is used here to wait for the stabilization of the material temperature
+after the setpoint ramp is complete.
+
+This also includes a simulated material that can be heated/cooled and a
+PID controller that can be connected to arbitrary systems. The PIDController
+has a ramp feature so that the setpoint ramps to the target value gradually.
+
+This example has two non-standard dependencies: ophyd, and simple_pid
+"""
+
+import asyncio
+import contextvars
+import functools
+import threading
+import time
+
+from collections import deque
+from simple_pid import PID
+from caproto.server import PVGroup, ioc_arg_parser, pvproperty, run, get_pv_pair_wrapper
+from ophyd import EpicsSignal, EpicsSignalRO, PVPositionerPC
+from ophyd import Component as Cpt
+
+
+pvproperty_with_rbv = get_pv_pair_wrapper(setpoint_suffix="", readback_suffix="_rbv")
```

Comment:
You found the secret `pvproperty_with_rbv` / `get_pv_pair_wrapper` - but you didn't use it below. Probably should remove this line